src/fab.ts

```diff
@@ -0,0 +1,8 @@
+import { Button } from '@bigtest/interactor';
+
+export default Button.extend('button')
+  .selector('button.MuiFab-root')
+  .filters({
+    ariaLabel: (el) => el.getAttribute("aria-label"),
+    svgIcon: (el) => el.querySelector('svg')?.classList.contains('MuiSvgIcon-root')
```

Comment:
It might be better to cast this to a boolean. If the SVG element is not found this would return `undefined`.

```suggestion
    svgIcon: (el) => !!el.querySelector('svg')?.classList.contains('MuiSvgIcon-root')
```